lib/evaluation/evaluationClientWrapper.ts

```diff
@@ -23,9 +26,85 @@ export class EvaluationClientWrapper {
   public getEvaluationClient() {
     return this._client;
   }
+  public selectFrom(
+    presentationDefinition: PresentationDefinition,
+    selectedCredentials: VerifiableCredential[],
+    did: string
+  ): SelectResults {
+    this._client.evaluate(
+      presentationDefinition,
+      new VP(new Presentation([], null, [], selectedCredentials, null)),
+      did
+    );
 
-  public evaluate(pd: PresentationDefinition, vp: VerifiablePresentation): EvaluationResults {
-    this._client.evaluate(pd, vp);
+    const marked: HandlerCheckResult[] = this._client.results.filter(
+      (result) =>
+        result.evaluator === 'MarkForSubmissionEvaluation' && result.payload.group && result.status !== Status.ERROR
+    );
+    this.evaluateRequirements(presentationDefinition.submission_requirements, marked, 0);
+    return {
+      matches: [...this.matchSubmissionRequirements(presentationDefinition.submission_requirements, marked)],
+      warnings: [],
```

Comment:
don't we want to put the existing warnings in here?

Reply:
Added warnings to the array.